lib/ktmw32-sys/build.rs

```diff
@@ -1,5 +1,19 @@
 // Copyright © 2015, Peter Atashian
 // Licensed under the MIT License <LICENSE.md>
+use std::env::var;
+
+fn link_win32_lib(libname: &str) {
```

Comment:
This should probably be common for all `*-sys` crates, but that would require another build dependency.